packages/project-access/src/types/mta/index.ts

```diff
@@ -0,0 +1,7 @@
+export interface MtaPath {
+    // File path that contains mta.yaml file
+    mtaPath: string;
+    // `hasRoot` is true if mta.yaml is in a parent folder (managed MTA project).
+    // It is set to false if mta.yaml is at in the project root folder (standardalone MTA project).
```

Comment:
Different folder structures that can support mta.yaml;

managed approuter (1-1)
```
- fioiruiapp
- fioiruiapp -> mta.yaml
- fioiruiapp -> package.json
```

managed approuter (1-*)
```
- rootfolder
- rootfolder -> mta.yaml
- rootfolder -> package.json
- rootfolder -> fioriouiapp1 -> package.json
- rootfolder -> fioriouiapp2 -> package.json
```

standalone approuter
```
- rootfolder
- rootfolder -> mta.yaml
- rootfolder -> package.json
- rootfolder -> approuter -> package.json
- rootfolder -> fioriouiapp1 -> package.json
``

Reply:
@longieirl I think all scenarios are covered. Do you have any suggestions on the return type structure?

Reply:
Managed approuter (1-*) is the same structure as standalone, so the only difference will be in the mta configuration where standalone will have a `approuter.nodejs`.